packages/now-cli/test/dev/integration.js

```diff
@@ -53,11 +53,23 @@ async function exec(directory, args = []) {
   });
 }
 
+async function runNpmInstall(fixturePath, t) {
+  if (await fs.exists(path.join(fixturePath, 'package.json'))) {
+    if (t) {
+      t.log(`Install dependencies for "${fixturePath}"`);
+    }
+
+    return execa('yarn', ['install'], { cwd: fixturePath, reject: false });
```

Comment:
Perhaps `reject` should be a parameter because most of the tests you probably want to fail when yarn install fails.